.azure-pipelines/azure-pipelines.yml

```diff
@@ -106,6 +106,7 @@ jobs:
       VERSION="$(cd $BUILD_SOURCESDIRECTORY && agvtool vers -terse)"
       [[ $BUILD_SOURCEBRANCH != 'refs/heads/master' ]] && VERSION="$VERSION+$(cd $BUILD_SOURCESDIRECTORY && git rev-parse --short $BUILD_SOURCEVERSION)"
       "$BUILD_SOURCESDIRECTORY/Scripts/create-archive.sh" "PLCrashReporter-$VERSION" "iOS Framework" "tvOS Framework" "Mac OS X Framework" "Tools"
+      "$BUILD_SOURCESDIRECTORY/Scripts/create-archive.sh" "PLCrashReporter-$VERSION.carthage.framework" "iOS Framework" "tvOS Framework" "Mac OS X Framework" "Tools"
```

Comment:
Should we add this in Xcode as well?

<img width="587" alt="Screenshot 2021-06-09 at 11 39 43" src="https://user-images.githubusercontent.com/536497/121322141-658b7080-c917-11eb-90e5-99f1df6bd8dd.png">